### PR TITLE
ConsoleInteraction: Sort settings by name

### PR DESCRIPTION
**Pull request by SanketDG**
Fixes https://github.com/coala-analyzer/coala/issues/2652

### CODE REVIEW

File: coalib/output/ConsoleInteraction.py [hunk 1]
```diff
@@ -505,7 +505,7 @@ def acquire_settings(log_printer, settings_names_dict):
 
     result = {}
     for setting_name, arr in sorted(settings_names_dict.items(),
-                                    key=lambda x: join_names(x[1][1:])):
+                                    key=lambda x: (join_names(x[1][1:]), x[0])):
```
Comment (Makman2) [inline coalib/output/ConsoleInteraction.py]: I think this won't change order as the comparison for `join_names(x[1][1:])` still has priority^^ (though I'm not sure what's inside `arr`...)

Comment (SanketDG) [inline coalib/output/ConsoleInteraction.py]: Yes that is what I wanted, `x[1][1:]` has the bear name, so I want it to be the first key, and then alphabetically sort the settings of that bear, so `x[0]`

Comment (Makman2) [inline coalib/output/ConsoleInteraction.py]: aaahhh alright :D

**Comment by Makman2** [comment]
ack 0030518

**Comment by SanketDG** [comment]
reack 57aecc8

**Comment by SanketDG** [comment]
reack 9aed540

File: coalib/output/ConsoleInteraction.py (hunk 1 above)
Comment (sils1297) [inline coalib/output/ConsoleInteraction.py]: can we file an issue to make this dict a namedtuple? This is super confusing.

**Comment by sils1297** [comment]
rebase needed again, forgot to merge?

**Comment by SanketDG** [comment]
reack 7d8a0ad

**Comment by SanketDG** [comment]
sorry I keep forgetting about this. :smile:

**Comment by SanketDG** [comment]
@rultor merge

**Pull request merged**